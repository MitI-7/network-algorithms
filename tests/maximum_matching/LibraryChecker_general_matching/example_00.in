7 8
2 0
0 5
5 6
6 1
1 0
1 3
3 4
1 4
