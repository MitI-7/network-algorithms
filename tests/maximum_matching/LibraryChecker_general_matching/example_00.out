3
0 2
1 6
3 4
